exercises/object-manipulation-2/src/defaults.js

```diff
@@ -0,0 +1,16 @@
+/* exported defaults */
+function defaults(target, source) {
+  debugger
+  for (var keySource in source) {
+    truth = false;
+    for (var keyTarget in target) {
```

Comment:
This `for...in` loop is unnecessary, the `in` operator in your conditional will check if the property exists without a need for a loop